src/libraries/System.Text.Json/src/readme.md

```diff
@@ -0,0 +1,54 @@
+## About
+
+Provides high-performance and low-allocating types that serialize objects to JavaScript Object Notation (JSON) text and deserialize JSON text to objects, with UTF-8 support built-in. Also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM), that is read-only, for random access of the JSON elements within a structured view of the data.
```

Comment:
We should mention `JsonSerializer` since it's the highest-level type and also that it provides native async functionality.